pkg/naming/constants.go

```diff
@@ -132,3 +142,17 @@ const (
 const (
 	ScyllaRuntimeConfigKey string = "ScyllaRuntimeConfig"
 )
+
+const (
+	PortNameCQL              = "cql"
```

Comment:
these seem to be limited to the desired service definition and not used elsewhere - do we want to promote them for reuse or rather inline them?

Reply:
done, moved to unexported consts in resource.go